specification/search/data-plane/Azure.Search/preview/2019-05-06-preview/searchservice.json

```diff
@@ -5773,13 +5773,238 @@
         },
         "configuration": {
           "type": "object",
+          "properties": {
+            "parsingMode": {
+              "type": "string",
+              "required": false,
+              "$ref": "#/definitions/ParsingMode"
+            },
+            "excludedFileNameExtensions": {
+              "type": "string",
+              "required": false,
+              "description": "Comma-delimited list of filename extensions to ignore when processing from Azure blob storage.  For example, you could exclude \".png, .mp4\" to skip over those files during indexing."
+            },
+            "indexedFileNameExtensions": {
```

Comment:
Maybe we should have default values defined for all optional properties?

Reply:
In general, I agree, but with open strings like here in excludedFileNameExtensions and indexedFileNameExtensions, there isn't really a default.  No file name extensions are excluded or included if you don't use them.

Reply:
I did not provide defaults for all open strings, but I did add a default for queryTimeout in commit https://github.com/Azure/azure-rest-api-specs/pull/9900/commits/a589333b19f4a75fde19b1d8556057c221b32fc8.